ddl/ddl_worker.go

```diff
@@ -97,6 +97,23 @@ func (d *ddl) getFirstDDLJob(t *meta.Meta) (*model.Job, error) {
 	return job, errors.Trace(err)
 }
 
+// handleTooLargeJob handles the too large DDL job.
+func (d *ddl) handleTooLargeJob(t *meta.Meta, job *model.Job, err error) error {
```

Comment:
May be `handleUpdateJobError` is better, probably other errors need to be handled in the future.